modules/apps/object/object-rest-test/src/testFunctional/tests/NestedFieldsDepth.testcase

```diff
@@ -32,7 +32,7 @@ definition {
 				name = "University");
 		}
 
-		task ("And Given studentUsers, usersUniversities, universitySubjects, subjectsUsers, userBooks, booksAuthors") {
+		task ("And Given studentUsers, usersUniversities, universitySubjects, subjectsUsers, userBooks, booksAuthors, userStudents, studentsUniversities") {
```

Comment:
in the end why do we put the userStudents, studentsUniversities here? I think it should be removed, it's an inconsistency from the other test, but in the end these relationships are not needed

Reply:
All the new cases need these two relationships